x-pack/legacy/plugins/siem/public/components/ml_popover/ml_modules.tsx

```diff
@@ -9,4 +9,10 @@
  * Detection UI. Added as part of: https://github.com/elastic/kibana/pull/39678/files
  *
  */
-export const mlModules: string[] = ['siem_auditbeat', 'siem_winlogbeat'];
+export const mlModules: string[] = [
+  'siem_auditbeat',
+  'siem_auditbeat_auth',
+  'siem_packetbeat',
+  'siem_winlogbeat',
+  'siem_winlogbeat_auth',
+];
```

Comment:
Compared this list to tests here:
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/ml/server/models/data_recognizer/__tests__/data_recognizer.js#L25

And the list on chat logs and that looks to be the correct ones from what I can tell. Approving